modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -505,12 +565,13 @@ impl ServerModule for Wallet {
         {
             return Err(WalletInputError::PegInAlreadyClaimed);
         }
-
+        let amount = fedimint_core::Amount::from_sats(input.tx_output().value);
+        let fee = self.cfg.consensus.fee_consensus.peg_in_abs;
+        WALLET_PEGIN_AMOUNTS_SATS.observe(amount.sats_f64());
+        WALLET_PEGIN_FEES_AMOUNTS_SATS.observe(fee.sats_f64());
+        WALLET_PEGIN.inc();
```

Comment:
AFAIR histograms already track total count, so this one is not necessary.

Reply:
![image](https://github.com/fedimint/fedimint/assets/163706/71144e67-5f36-49ff-8bfb-13f49b9b84df)